### PR TITLE
Finish HCA Javascript port.

### PR DESCRIPTION
**Pull request by awong-dev**

### CODE REVIEW

**Comment by akainic** [comment]
Where are the test files going to live? I see only one e2e test file in the test directory for HCA.

**Comment by awong-dev** [comment]
oops. see https://github.com/department-of-veterans-affairs/vets-website/pull/2787

File: src/js/common/index.js [hunk 1]
```diff
@@ -5,9 +5,6 @@
 // other such libraries in here. Most of the site does not use these legacy
 // frameworks and it belongs in a lower-level module.
 
-// Global site style.
-require('../../sass/style.scss');
```
Comment (akainic) [inline src/js/common/index.js]: @webinista I'm finally getting to merging this PR, and I think this small change of moving the import of `style.scss` to `no-react-entry.js` is fixing the problem of the USWDS being imported twice. Does this change look correct to you?

**Comment by awong-dev** [comment]
HCA is not supposed to load no-react-entry.js. .. if it is, there's
something wrong in the metalsmith template.

File: src/js/common/index.js (hunk 1 above)
Comment (webinista) [inline src/js/common/index.js]: @akainic I'm the exact wrong person to ask this question. But that sounds like a good fix to me. Moving it out of `index.js` definitely solved the problem of double imports. Moving it to `no-react-entry.js` is probably the correct fix.

Comment (akainic) [inline src/js/common/index.js]: Awesome, I tested it locally and it seemed to fix things for me as well.

**Comment by akainic** [comment]
@awong-dev HCA is not loaded no-react-entry.js, we were seeing a separate problem where the styles were being loaded twice, and moving them to no-react-entry.js seems to solve this problem.

File: src/js/common/index.js (hunk 1 above)
Comment (akainic) [inline src/js/common/index.js]: @crwallace and @bshyong can you both also take a look at this change and confirm that moving the import of `style.scss` from `index.js` to `no-react-entry.js` works?

Comment (bshyong) [inline src/js/common/index.js]: @akainic I'm not sure of the root cause for the double imports, but moving `style.scss` from `index.js` to `no-react-entry.js` looks fine—the `no-react-entry.js` requires `index.js` either way

https://github.com/department-of-veterans-affairs/vets-website/blob/fix-js/src/js/no-react-entry.js#L4

**Comment by webinista** [comment]
This should resolve #3057 once merged.

**Pull request merged**